service/matching/matchingEngine.go

```diff
@@ -323,9 +323,9 @@ func (e *matchingEngineImpl) AddWorkflowTask(
 	// We don't need the userDataChanged channel here because:
 	// - if we sync match or sticky worker unavailable, we're done
 	// - if we spool to db, we'll re-resolve when it comes out of the db
-	taskQueue, _, err := e.redirectToVersionedQueueForAdd(ctx, origTaskQueue, addRequest.VersionDirective, stickyInfo)
```

Comment:
instead of returning it, I'd rather have callers get the unversioned tqm and pass it into redirectToVersionedQueueForAdd. actually I already made that change in a pending PR (not ready for review yet)

also, I think calling it `baseTqm` is a little nicer